stacks-signer/src/client/stacks_client.rs

```diff
@@ -644,8 +648,8 @@ impl StacksClient {
         address: &StacksAddress,
     ) -> Result<AccountEntryResponse, ClientError> {
         debug!("stacks_node_client: Getting account info...");
-        let timer =
-            crate::monitoring::new_rpc_call_timer(&self.accounts_path(address), &self.http_origin);
+        let timer_label = format!("{}/v2/accounts/:stacks_address", self.http_origin);
```

Comment:
[The Stacks Core metrics use `:principal` for the same endpoint](https://github.com/stacks-network/stacks-core/blob/master/stackslib/src/net/api/getaccount.rs#L88). Should we align and use `:principal` here too? Or perhaps `:address` is more correct?

Reply:
Good call, changed!